openreview/venue/process/desk_rejected_submission_process.py

```diff
@@ -51,4 +49,32 @@ def process(client, edit, invitation):
         )
 
     print(f'Remove {paper_group_id}/{authors_name} from {venue_id}/{authors_name}')
-    client.remove_members_from_group(f'{venue_id}/{authors_name}', f'{paper_group_id}/{authors_name}')
\ No newline at end of file
+    client.remove_members_from_group(f'{venue_id}/{authors_name}', f'{paper_group_id}/{authors_name}')
+
+    recipients = [f'{paper_group_id}/{authors_name}']
```

Comment:
@melisabok here we were always emailing all committee for a desk-rejection; I changed it to emailing only the readers like we do in the withdrawal process function.

Reply:
I think we still need to check if the group exists. because the readers can be SubmissionX/Senior_Area_Chairs and the group hasn't been created yet

Reply:
yes! I am still checking if the groups exist